tests/test_examples.py

```diff
@@ -52,7 +52,8 @@ def test_run_tutorial_notebooks(nb_path) -> None:  # pragma: no cover
         nb_path: Path to the notebook to test.
     """
     nb = ptnb.notebook.load_notebook(nb_path)
-    result = ptnb.execution.execute_notebook(nb, cwd=TUTORIALS_DIR, timeout=120)
+    # TODO: Shorten timeout and ensure the notebook can still show desired improvement.
```

Comment:
Following https://google.github.io/styleguide/pyguide.html#312-todo-comments we suggest including username or GH issue in TODO comments so that future developers can find more information about it as needed. I think you already opened issue #793  to track this so can reference that

Reply:
Done.